### PR TITLE
feat: Add hover effect to Navbar links

### LINKED ISSUE #144: Adding hover effect to Navbar links
### What feature do you want to see added?

Navbar links don't change their style on hover.
Screenshot from 2024-03-17 19-20-39

### Upstream changes

_No response_

### PR DESCRIPTION
**Pull request by Ratul-Saha-003**
Added hover effect to Navbar links.

**Before:**
Screenshot from 2024-03-17 19-20-39

**After:**
Screenshot from 2024-03-17 19-23-05

Fixes #144

### CODE REVIEW

File: src/jio-navbar.css [hunk 1]
```diff
@@ -104,19 +104,27 @@ button:not(:disabled) {
   line-height: inherit;
   margin: 0 auto;
 }
+/* .nav-link.dropdown-toggle:hover {
```
Comment (halkeye) [inline src/jio-navbar.css:107]: What's the purpose of all this commented out code?

File: src/jio-navbar.css [hunk 2]
```diff
@@ -104,19 +104,27 @@ button:not(:disabled) {
...
 
 button.nav-link {
   background: #f9f9f9;
   border: 1px solid #ccc;
   border-radius: 3px;
   box-shadow: 0 2px 4px -3px rgb(0 0 0 / 25%);
 }
-
-.nav-link:hover{
+button.nav-link:hover {
```
Comment (halkeye) [inline src/jio-navbar.css:118]: Is there a reason you are specifically targetting buttons now?

**Comment by Ratul-Saha-003** [comment]
@halkeye The PR was incomplete. It is complete now. I can't understand the reason for this linting error.

**Comment by halkeye** [comment]
>     <file name="/home/jenkins/agent/workspace/tes_jenkins-io-components_PR-145/src/jio-navbar.css">
>      <error source="stylelint.rules.rule-empty-line-before" line="107" column="1" severity="error" message="Expected empty line before rule (rule-empty-line-before)" />
>    </file>

does line 107 on jio-navbar.css have a empty line before the next rule?

**Comment by Ratul-Saha-003** [comment]
> > ```
> > <file name="/home/jenkins/agent/workspace/tes_jenkins-io-components_PR-145/src/jio-navbar.css">
> >  <error source="stylelint.rules.rule-empty-line-before" line="107" column="1" severity="error" message="Expected empty line before rule (rule-empty-line-before)" />
> > ```
> 
> does line 107 on jio-navbar.css have a empty line before the next rule?


Made the appropriate changes.

**Comment by halkeye** [comment]
I'll admit, i'm not sure why a hover effect is wanted. Have you talked to any of the sig leaders about it? like @MarkEWaite or @kmartens27 ?

**Comment by Ratul-Saha-003** [comment]
No, but I was working with the Navbar component when I noticed that it is very easy to click another link when there is no difference on hover.

**Review by krisstern — APPROVED** [review]
Thanks @Ratul-Saha-003 for the PR!

**Pull request merged**